docs/src/main/asciidoc/building-native-image.adoc

```diff
@@ -14,7 +14,54 @@ This guide covers:
 
 This guide takes as input the application developed in the link:getting-started[Getting Started Guide].
 
-== Prerequisites
+== GraalVM
+
+Building a native executable requires using a distribution of GraalVM.
+There are three distributions:
+Oracle GraalVM Community Edition (CE), Oracle GraalVM Enterprise Edition (EE) and Mandrel.
+The differences between the Oracle and Mandrel distributions are as follows:
+
+* Mandrel is a downstream distribution of the Oracle GraalVM CE.
+Mandrel's main goal is to provide a way to build native executables specifically designed to support Quarkus.
+
+* Mandrel releases are built from a code base derived from the upstream Oracle GraalVM CE code base,
+with only minor changes but some significant exclusions.
+They support the same capabilities to build native executables as Oracle GraalVM CE,
+with no significant changes to functionality.
+Notably, they do not include support for polyglot programming.
+
+* Mandrel is built slightly differently to Oracle GraalVM CE, using the standard OpenJDK project.
+This means that it does not profit from a few small enhancements that Oracle have added to the version of OpenJDK used to build their own GraalVM downloads.
+
+* Mandrel is only available for projects that can be compiled with Java 11.
+If you are using Java 8,
+you should consider using Oracle GraalVM instead.
+
+* Mandrel is currently only recommended for building native executables that target Linux containerized environments.
+This means that Mandrel users should use containers to build their native executables.
+If you are building native executables for macOS or Windows target platforms,
+you should consider using Oracle GraalVM instead,
+because Mandrel does not currently target these platforms.
+Building native executables directly on bare metal Linux is possible,
+with details available in the https://github.com/graalvm/mandrel/blob/master/README-Mandrel.md[Mandrel README]
+and https://github.com/graalvm/mandrel/releases[Mandrel releases].
+
+The prerequisites vary slightly depending on whether you are using Oracle GraalVM CE/EE or Mandrel.
+
+== Prerequisites for Mandrel
+
+To complete this guide using Mandrel, you need:
+
+* less than 15 minutes
+* an IDE
+* JDK 11 installed with `JAVA_HOME` configured appropriately
+* A working container runtime (Docker, podman)
+* The code of the application developed in the link:getting-started[Getting Started Guide].
+
+Skip to the <<#container-runtime,this section>> to continue with the guide for Mandrel,
```

Comment:
I think this should be 
```suggestion
Skip to <<#container-runtime,this section>> to continue with the guide for Mandrel,
```

Reply:
It's fine as is, the text on the right is what is shown.

Reply:
Exactly, so the end result will be `Skip to the this section to...` while I think it should be `Skip to this section to...`

Reply:
Ah yes, you're suggesting removing `the`. Thought you meant the opposite :)